types/merge-deep/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for merge-deep 3.0
+// Project: https://github.com/jonschlinkert/merge-deep
+// Definitions by: Emily Marigold Klassen <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/merge-deep/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me